lib/linguist/languages.yml

```
@@ -227,6 +227,14 @@ Ada:
   tm_scope: source.ada
   ace_mode: ada
   language_id: 11
+Adblock:
+  type: markup
```

```suggestion
  type: data
```
`markup` refers to languages like HTML. The type should be `data` if it's just a text list of rules.